code/modules/antagonists/traitor/contractor/items/contractor_baton.dm

```diff
@@ -0,0 +1,27 @@
+/obj/item/melee/classic_baton/telescopic/contractor
+	name = "contractor baton"
+	desc = "A compact, specialised baton available to Syndicate contractors. Applies light electrical shocks to targets."
+	// Overrides
+	affect_silicon = TRUE
+	stun_time = 6 SECONDS_TO_LIFE_CYCLES
+	cooldown = 5 SECONDS
+	force_off = 5
+	force_on = 15
+	item_state_on = "contractor_baton"
+	icon_state_off = "contractor_baton_0"
+	icon_state_on = "contractor_baton_1"
+	stun_sound = 'sound/weapons/contractorbatonhit.ogg'
+	extend_sound = 'sound/weapons/contractorbatonextend.ogg'
+	// Settings
+	/// Stamina damage to deal on stun.
+	var/stamina_damage = 40
```

Comment:
suggestion: set cooldown to 2.5 sec, stun time to 1 sec and stamina damage 70.
why? mixes it up a bit, it's now a two hit to stamina crit instead of 4, makes it very hard to counter with chems. and allows for a very slim chance to fight back one on one since standing back up isn't affected by stamina.